addons/binding/org.openhab.binding.airquality/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,164 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="airquality"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- Air Quality Thing -->
+    <thing-type id="aqi">
+        <label>Air Quality</label>
+        <description>Provides various air quality data from the World Air Quality Project.
+        In order to receive the data, you must register an account on http://aqicn.org/data-platform/token/ and get
+        your API token.
+        </description>
+
+        <channels>
+            <channel id="aqiLevel" typeId="aqiLevel" />
+            <channel id="aqiDescription" typeId="aqiDescription" />
+            <channel id="pm25" typeId="pm25" />
+            <channel id="pm10" typeId="pm10" />
+            <channel id="o3" typeId="o3" />
+            <channel id="no2" typeId="no2" />
+            <channel id="co" typeId="co" />
+            <channel id="locationName" typeId="locationName" />
+            <channel id="locationUrl" typeId="locationUrl" />
+            <channel id="observationTime" typeId="observationTime" />
+            <channel id="stationId" typeId="stationId" />
+            <channel id="temperature" typeId="temperature" />
+            <channel id="pressure" typeId="pressure" />
+            <channel id="humidity" typeId="humidity" />
+        </channels>
+
+        <config-description>
+            <parameter name="apikey" type="text" required="true">
+                <context>password</context>
+                <label>API Key</label>
+                <description>Data-platform token to access the AQIcn.org service</description>
+            </parameter>
+            <parameter name="location" type="text" required="true" 
+            pattern="^[-+]?([1-8]?\d(\.\d+)?|90(\.0+)?)[,|;]\s*[-+]?(180(\.0+)?|((1[0-7]\d)|([1-9]?\d))(\.\d+)?)$">
+                <label>Location</label>
+                <description>Your geo coordinates separated with comma or semicolon (e.g. "37.8;-122.4").</description>
+            </parameter>
+            <parameter name="refresh" type="integer" min="30" required="false">
+                <label>Refresh interval</label>
+                <description>Specifies the refresh interval in minutes.</description>
+                <default>60</default>
+            </parameter>
+        </config-description>
+    </thing-type>
+
+    <channel-type id="aqiLevel">
+        <item-type>Number</item-type>
+        <label>Air Quality Index</label>
+        <description></description>
+        <category>Air Quality Index</category>
+        <state readOnly="true" pattern="%d"></state>
+    </channel-type>
+
+    <channel-type id="aqiDescription">
```

Comment:
According to the [website](http://aqicn.org/scale/), this value is referred to as "scale"  or "level". Channel id could perhaps better match this, so instead of `aqiDescription` use `aqiLevel`. Please note: you are using level for the actual index.
In my opinion an index is a value where a level is a range of values.